### PR TITLE
output logs when e2e case fails

### PR DESCRIPTION
**Pull request by zhiweiyin318**
output logs when e2e case fails for debug.

### CODE REVIEW

File: test/e2e/common.go [hunk 1]
```diff
@@ -425,3 +427,69 @@ func (t *Tester) GetClusterNameFromKlusterlet(klusterletName string) (string, er
...
+	if err != nil {
+		klog.Errorf("failed to list klusterlets. error: %v", err)
+	}
+	for _, klusterlet := range klusterletes.Items {
+		klog.Infof("klusterlet %v : %#v \n", klusterlet.Name, klusterlet)
+	}
+
+	managedClusters, err := t.ClusterClient.ClusterV1().ManagedClusters().List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		klog.Errorf("failed to list mangedClusters. error: %v", err)
```
Comment (pmorie) [inline test/e2e/common.go]: typo: managed

File: test/e2e/common.go [hunk 2]
```diff
@@ -425,3 +427,69 @@ func (t *Tester) GetClusterNameFromKlusterlet(klusterletName string) (string, er
...
+		klog.Infof("klusterlet agent pod %v/%v : %#v \n", pod.Namespace, pod.Name, pod)
+		logs, err := t.PodLog(pod.Name, pod.Namespace, int64(10))
+		if err != nil {
+			klog.Errorf("failed to get pod %v/%v log. error: %v", pod.Namespace, pod.Name, err)
+			continue
+		}
+		klog.Infof("pod %v/%v logs:\n %v \n", pod.Namespace, pod.Name, logs)
+	}
+
+	manifestWorks, err := t.WorkClient.WorkV1().ManifestWorks("").List(context.TODO(), metav1.ListOptions{})
```
Comment (pmorie) [inline test/e2e/common.go:479]: This works for now, but you may want to track specific namespaces (ie, only dump from the namespaces involved in the tests) in situations where the e2e is running against a live system in a support situation in the future.

**Review by pmorie — COMMENTED** [review]
Fix the typo and we'll land this. Thanks for the PR!

File: test/e2e/common.go (hunk 1 above)
Comment (zhiweiyin318) [reply test/e2e/common.go]: done

File: test/e2e/common.go (hunk 2 above)
Comment (zhiweiyin318) [reply test/e2e/common.go:479]: I add a TODO here, and will refactor this when I find a proper way to track the resources created during e2e.

**Pull request merged**